en/Useful-Websites-to-Learn-How-to-Code-Quickly.md

```diff
@@ -0,0 +1,95 @@
+#  Website recommendations for developers to learn programming 
+
+* en：[9 Useful Websites to Learn How to Code Quickly](http://www.queness.com/post/10709/8-useful-websites-to-learn-how-to-code-quickly)
+* zh translation:  [wangguo](http://wangguo.iteye.com/)
+
+The Internet is a rapidly evolving field, where technologies constantly being updated, like from HTML to HTML5, from CSS to CSS3, from JavaScript to JavaScript frameworks, etc. If you can't keep up with the speed, you will be out of game. Therefore, a quick grasp of a language or a technology can be very beneficial. 
+
+This article provides you 9 practical online tutorials, some in interactive format, and some in comprehensive guides and visualizations to help you quickly master a language. They share a common purpose, that is, make your learning more fun, and make it easier for you to master it.
+
+##  [LifeHacker Learn to code](http://www.lifehacker.com.au/2011/02/learn-to-code-the-full-beginners-guide/)
+
+Language: JavaScript 
+4 lessons plus best practices/resources to teach you how to program using Javascript. 
+
+Each lesson has a video and detailed articles to make sure you understand them.  
```

Comment:
Each lesson has a video and detailed articles to make sure that you understand them.